bedrock/firefox/templates/firefox/new/onboarding/scene1.html

```diff
@@ -117,10 +131,20 @@ <h2 class="forward">{{_('Go Forward') }}</h2>
 
   <div id="other-platforms">
     <div class="container">
-      <section class="section-other-platforms">
-        <h4>{{_('Firefox for other platforms') }}</h4>
+      {#
+        when l10n tag 'firefox_new_install_options' is removed, remove
+        'no-lang-link' class below and from the CSS file
+      #}
+      <section class="section-other-platforms{% if l10n_has_tag('firefox_new_install_options') %} no-lang-link{% endif %}">
+        <h4>
+        {% if l10n_has_tag('firefox_new_install_options') %}
+          {{ _('Advanced Install Options & Other Platforms') }}
+        {% else %}
+          {{_('Firefox for other platforms') }}
+        {% endif %}
+        </h4>
 
-        {{ download_firefox_desktop_list() }}
+        {{ download_firefox_desktop_list('release') }}
```

Comment:
Not sure if this is required as it should be the default value anyway, but should this be `channel='release'`?